### ISSUE TITLE
MCF format: Missing categories

### ISSUE THREAD

**Issue by luke-jr** [delight-im/MovieContentFilter#17]
Noticed some categories missing that could be useful:

* Superstition (probably with subcategories like tarot cards, fortune tellers, etc)
* Religion (per-religion subcategories? might need to support sub-subcategories?)
* Implied sexual relationships (especially homosexuality; some might even prefer to exclude all same-sex relationships of a certain affinity, even if not sexual in nature?)
* Sex subcategories for contraception and birth control
* Violence subcategory for abortion
* Identification (it can be fun to have a surprise movie where some viewers do not see the title upfront)

**Comment by ocram** [comment]
Thank you very much!

While it’s certainly subjective what is considered content that should be filtered or even just content that people might want to filter, I think we can probably agree that several of your suggested categories would be useful for large groups of people (that *personally* decide they want to filter that content, while it might be fine for other people, as always):

 * Homosexuality: Sure, some people might want to block this. But what about bisexuality, asexuality, or even heterosexuality then? Why should people be able to block one orientation while not the others? Should we just add all of them? And may people need specific types of homosexuality as categories? Don’t know. Certainly under the “Sex” category, because if it’s just two men or two women talking, why would you block this … It’s probably only when their relationship becomes sexual that your religious beliefs (or something else) makes you want to avoid this.
 * Contraception and birth control: Definitely, under the “Sex” category, as you suggested. But would that be *two* categories? It’s just that contraception is more specific than birth control, right? So would the latter be sufficient?
 * Abortion: Certainly, and yes, probably under “Violence” (because where else).

Apart from that, there are some categories that need more discussion perhaps:

 * Religion and superstition: Yes, we should probably add these two as well. They don’t seem to fit under any existing category, so perhaps “Beliefs” as a new parent category? As for any subcategories, somebody more knowledgeable may need to make suggestions here. As for subcategories for specific religions: Why would you block these? Wouldn’t just be a lack of tolerance, unless your own religion doesn’t allow you to accept other religions? On the other hand, if you can filter specific forms of superstition …
 * Identification: In itself this is probably too broad. But with subcategories for the identification of the movie’s title (or other information), this might certainly make sense.

**Comment by luke-jr** [comment]
>Homosexuality: Sure, some people might want to block this. But what about bisexuality, asexuality, or even heterosexuality then? Why should people be able to block one orientation while not the others? Should we just add all of them? 

Celibacy and properly ordered sexual relationships are normal. Can't hurt to have them all as categories, I suppose, but you'd probably end up with very little content left if you removed all normal relationships.

> Contraception and birth control: Definitely, under the “Sex” category, as you suggested. But would that be two categories? It’s just that contraception is more specific than birth control, right? So would the latter be sufficient?

Maybe.

> As for subcategories for specific religions: Why would you block these? Wouldn’t just be a lack of tolerance, unless your own religion doesn’t allow you to accept other religions? 

Most religions do not consider other religions to be acceptable, or at least not true.

**Comment by ocram** [comment]
The point is that what’s considered normal and what isn’t varies between communities, religions, countries, etc. This project here does not want to judge, but instead wants to provide the means for *all* such communities (or at least most of them) to filter what *they* want filtered.

But we seem to agree in that adding the possibility to block those other relationships cannot hurt, and nobody will block *all categories* anyway, so you will never have the problem that there’s no content left because you’re blocking everything.

While most religions don’t consider other religions true (of course), fortunately most of those religions also don’t see other people believing in other religions as something that you must not see or hear about or something you must not tolerate. But again, this is not what the focus of this project is on. We want to give people what they need to block what they want blocked.

**Comment by luke-jr** [comment]
> nobody will block all categories anyway, so you will never have the problem that there’s no content left because you’re blocking everything.nobody will block all categories anyway, so you will never have the problem that there’s no content left because you’re blocking everything.

My point was that blocking all proper relationships will likely leave you with nothing left, because it's so commonplace.

> While most religions don’t consider other religions true (of course), fortunately most of those religions also don’t see other people believing in other religions as something that you must not see or hear about or something you must not tolerate.

Sure, but I was thinking details of those religions. For example, Christians shouldn't be exposing their young children to false ideas like reincarnation.

**Comment by ocram** [comment]
> My point was that blocking all proper relationships will likely leave you with nothing left, because it's so commonplace.

Sure, but nobody *has* to block those relationships (and thus all content), and, again, people just talking would not be classified here, because (while that is an interpersonal relationship) it does not depict a sexual relationship.

> Sure, but I was thinking details of those religions. For example, Christians shouldn't be exposing their young children to false ideas like reincarnation.

So if this is not about seeing people practice other religions, but instead about depictions and descriptions of ideas and concepts from other religions (that you might not want your children to see), that makes sense. But, if your religion is X, instead of offering to block religions A, B and C, I guess it would make more sense to just offer non-X (and non-A, non-B and non-C) to be blocked. Because you don’t want to block specific other religions that you don’t tolerate but simply all ideas that are not shared in your own religion.

**Comment by luke-jr** [comment]
>people just talking would not be classified here, because (while that is an interpersonal relationship) it does not depict a sexual relationship.

But marriage and children do.

> So if this is not about seeing people practice other religions, but instead about depictions and descriptions of ideas and concepts from other religions (that you might not want your children to see), that makes sense. 

Well, presumably seeing the practice of them might also be included in this.

> But, if your religion is X, instead of offering to block religions A, B and C, I guess it would make more sense to just offer non-X (and non-A, non-B and non-C) to be blocked. Because you don’t want to block specific other religions that you don’t tolerate but simply all ideas that are not shared in your own religion.

I think your approach is probably better.

**Comment by jacob-willden** [comment]
I'm not sure about everything discussed above, but I certainly think it would make sense to provide differentiation for at least some of the subcategories under the sex category, like with the kissing subcategory (e.g. heterosexual kissing, homosexual kissing, other(?) kissing). Not sure about the other subcategories though.

While I personally would not differentiate such subcategories in my own editing, I know many people would prefer such differentiation for a variety of reasons.

I completely agree that abortion and birth control should be added. I also think birth control by itself should be fine (no need to add contraception separately, in my opinion).

What does everyone else think?

**Comment by Ra0k** [comment]
> I'm not sure about everything discussed above, but I certainly think it would make sense to provide differentiation for at least some of the subcategories under the sex category, like with the kissing subcategory (e.g. heterosexual kissing, homosexual kissing, other(?) kissing). Not sure about the other subcategories though.
> 
> I completely agree that abortion and birth control should be added. I also think birth control by itself should be fine (no need to add contraception separately, in my opinion).
> 
> What does everyone else think?

I agree. Also, some people might want to filter e.g. homosexual wedding scenes while want to keep heterosexual. The same applies to kissing, hugging, romantic talk or just simply referring to the status of their relationship. Under what category can it be done now? @ocram

**Comment by jacob-willden** [comment]
I’ve been giving this topic a lot of thought the past few months, and I definitely think that a Mastodon-type approach will be best, meaning that we have one extension that can connect to one of many community-maintained servers (in addition to the official one) that are identical expect for additional filter categories and perhaps other small differences. Some servers can be more specific in their categories than others, and can have different criteria.
